app/config/Configuration.scala

```diff
@@ -27,13 +27,11 @@ class Configuration {
 
   lazy val paymentApiUrl = config.getString("paymentApi.url");
 
-  lazy val contributionsFrontendUrl = config.getString("contribution.url")
-
   lazy val membersDataServiceApiUrl = config.getString("membersDataService.api.url")
 
   lazy val goCardlessConfigProvider = new GoCardlessConfigProvider(config, stage)
 
-  lazy val payPalConfigProvider = new PayPalConfigProvider(config, stage)
+  lazy val payPalNvpConfigProvider = new PayPalConfigProvider(config, stage)
```

Comment:
I would leave this as `payPalConfigProvider`. I don't see that adding NVP to it adds any value.

Reply:
There are two PayPal integration, Nvp (directly integrated via the PayPal service) and Rest (indirectly integrated via the payment API Service). I added NVP to mark that is the NVP integration.